.github/workflows/ci.yml

```diff
@@ -78,6 +78,6 @@ jobs:
       - name: Install ansible
         run: pip install ansible
 
-      # - name: Update imports on Ansible Galaxy
-      #   run: |
-      #     ansible-galaxy role import --api-key "${{ secrets.GITHUB_TOKEN }}" openwisp ansible-openwisp2
+      - name: Update imports on Ansible Galaxy
+        run: |
+           ansible-galaxy role import --api-key ${{ secrets.GALAXY_API_KEY }} openwisp influxdb
```

Comment:
@nemesifier was these lines commented purposefully? You will need add the GALAXY_API_KEY to secrets of this repository.